influxdb/Makefile

```diff
@@ -69,7 +69,7 @@ push: gcr-login $(addprefix sub-push-,$(ALL_ARCHITECTURES))
 
 sub-push-%:
 	$(MAKE) ARCH=$* PREFIX=$(PREFIX) VERSION=$(VERSION) build
-	docker push $(PREFIX)/heapster-influxdb-$(ARCH):$(VERSION)
+	docker push $(PREFIX)/heapster-influxdb-$*:$(VERSION)
```

Comment:
yes, this was an oversight from earlier.
`$*` resolves to the `%` parameter, which is the arch being built. `$(ARCH)` in this case resolves statically to amd64 every time